lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -221,7 +228,7 @@ def video_summary(video_profiles, course_id, video_descriptor, request, local_ca
             },
             request=request,
         )
-        for lang in transcript_langs
+        for lang in set(transcript_langs)
```

Comment:
Why `set` again here?

Reply:
get_available_transcript_languages may return codes that are already in transcript_langs